includes/theme.inc

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * @file
+ * Helper functions to support theming in the Gent Base theme.
+ */
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Drupal\paragraphs\Entity\Paragraph;
+
+/**
+ * Returns a list of form_id's of the system forms.
+ *
+ * @return array
+ *   An array with form_id values.
+ */
+function _gent_base_get_system_forms() {
+  return [
+    'content_moderation_entity_moderation_form',
+    'views_exposed_form',
+  ];
+}
+
+/**
+ * Check to see if a given view mode exists.
+ *
+ * @param string $key
+ *   The view mode key.
+ *
+ * @return mixed
+ *   Returns the view mode if found.
+ */
+function gent_base_is_view_mode($key) {
+  return \Drupal::service('entity_type.manager')
+    ->getStorage('entity_view_display')
+    ->load($key);
+}
+
+/**
+ * Helper function to get the file data of a referenced File entity.
+ *
+ * @param mixed $variables
+ *   The variables array to work with.
+ * @param string $field
+ *   The field to get the referenced file file size from.
+ *
+ * @return array|null
+ *   The file size of the document in bytes.
+ *
+ * @throws \Drupal\Core\TypedData\Exception\MissingDataException
+ */
+function _gent_base_get_entity_reference_file_data(&$variables, $field) {
+  $data = [];
+
+  /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+  $paragraph = $variables['elements']['#paragraph'];
+  if (!$paragraph->hasField($field)) {
+    return;
+  }
+  /* @var $entity \Drupal\Core\Field\Plugin\Field\FieldType\EntityReferenceItem */
+  $entity = $paragraph->get($field)->first();
```

Comment:
Gaat altijd maar over 1 value? Nooit multi value?

Reply:
Wordt enkel gebruikt om `field_primary_action_document` en `field_secondary_action_document` op te halen, wat nooit meer dan 1 file zou mogen zijn (want dat is niet ondersteund)